cmd/nodecmd/whitelist.go

```diff
@@ -0,0 +1,170 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package nodecmd
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"golang.org/x/exp/slices"
+	"golang.org/x/net/context"
+
+	awsAPI "github.com/ava-labs/avalanche-cli/pkg/cloud/aws"
+	gcpAPI "github.com/ava-labs/avalanche-cli/pkg/cloud/gcp"
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanche-cli/pkg/models"
+	"github.com/ava-labs/avalanche-cli/pkg/utils"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+	"github.com/pingcap/errors"
+	"github.com/spf13/cobra"
+)
+
+func newWhitelistCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "whitelist <clusterName> [IP] ",
+		Short: "(ALPHA Warning) Whitelist IP for access to all nodes in a cluster",
+		Long: `(ALPHA Warning) This command is currently in experimental mode.
+
+The node whitelist command adds IP to cloud security access rules allowing it to access
+all nodes in the cluster via ssh or http.`,
+		SilenceUsage: true,
+		Args:         cobra.MinimumNArgs(1),
+		RunE:         whitelistIP,
+	}
+	return cmd
+}
+
+func whitelistIP(_ *cobra.Command, args []string) error {
```

Comment:
probably it makes sense for this command to call a function that is also used by node create, so as to centralize this firewall setting

Reply:
maybe.